addons/mail/models/discuss/discuss_channel_category.py

```diff
@@ -0,0 +1,30 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.addons.mail.tools.discuss import Store
+
+
+class ChannelCategory(models.Model):
+    _name = "discuss.channel.category"
+    _description = "Channel Category"
+
+    name = fields.Char('Channel category name', required=True)
+    channel_ids = fields.One2many('discuss.channel', 'channel_category_id', string='Channels')
```

Comment:
How are we going to sort them? Don't we need a sequence field?

I don't think id of name will be appropriate for this use case.